src/qt/receivecoinsdialog.h

```diff
@@ -36,7 +36,7 @@ class ReceiveCoinsDialog : public QDialog
     enum ColumnWidths {
         DATE_COLUMN_WIDTH = 130,
         LABEL_COLUMN_WIDTH = 120,
-        AMOUNT_MINIMUM_COLUMN_WIDTH = 160,
+        AMOUNT_MINIMUM_COLUMN_WIDTH = 180,
```

Comment:
Needs to be larger for me. (Or use the diff I provided below)

![screenshot from 2016-01-20 16-50-46](https://cloud.githubusercontent.com/assets/6399679/12454067/3644feb2-bf96-11e5-9c6f-dd0870013216.png)